tests/test_trx.py

```diff
@@ -466,7 +470,7 @@ def test_trx_sign_message(self, backend, firmware, navigator):
 
         with backend.exchange_async(CLA, InsType.SIGN_PERSONAL_MESSAGE, 0x00,
                                     0x00, data):
-            if firmware.device == "stax":
+            if not firmware.device.startswith("nano"):
```

Comment:
The test is easier to read, in reverse, without `not`:
```
if firmware.device.startswith("nano"):
    text = "message"
else:
    text = "Hold to sign"
```

Reply:
On top of that, it is possible to use `firmware.is_nano` boolean instead of this convoluted `firmware.device.startswith("nano")` thingy.

Reply:
Ok 👍